docs/simulation/calculations.yaml

```diff
@@ -92,4 +92,32 @@ POPs:
               owners: total size of owner POPs in state
               workers: number of employed workers in RGO
               max(0.5: Hardcoded minimum of half the RGO income.
-              2*: Hardcoded value.
\ No newline at end of file
+              2*: Hardcoded value.
+Military:
+  Reinforcements per army:
+    formula: 5 * round(supply factor * sum of unit reinforcements * REINFORCE_SPEED * local reinforce rate * (1 + reinforce_rate) * (1 + reinforce_speed) / 5)
+    supply factor:
+      if unit received all its supplies: 1
+      else: complex graph, see https://www.desmos.com/calculator/gu1zpo01qz
+    unit reinforcements: 1000 * max_strength * unit reinforce rate
+      max_strength: as defined in /units/<unit>.txt
+      1000: (max_)strength is defined in thousands of soldiers. Reinforcements per army uses the number of soldiers.
+      unit reinforce rate:
+        if unit is mobilised: 0.5
+        else if pop is too small for a single unit: 0.5
+        else if pop supports too many units: number of units it can support / number of supported units
```

Comment:
Is the number of units it can support `pop size` divided by `defines.military.POP_SIZE_PER_REGIMENT = 3000`? And does this mean that any unit requires `3000` pops, regardless of its `max_strength`?

Reply:
Yes. As above, different section imo.